package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.42",
+  "version": "1.1.45",
```

Comment:
The reason for bump from 42 to 45 is I've found yarn link and local development flaky, so I just prefer to bump the EDS patch versions until it's working properly ;)